src/parquet/column/writer.h

```diff
@@ -164,11 +177,14 @@ inline void TypedColumnWriter<DType>::WriteBatch(int64_t num_values,
   }
 
   if (num_rows_ > expected_rows_) {
-    throw ParquetException("More rows were written in the column chunk then expected");
+    throw ParquetException("More rows were written in the column chunk than expected");
   }
 
   WriteValues(values_to_write, values);
 
+  auto stats = static_cast<TypedRowGroupStatistics<DType>*>(page_statistics_.get());
+  stats->Update(values, values_to_write, num_values - values_to_write);
```

Comment:
Does it make sense to make writing the statistics optional? Can add an option to WriterProperties. I looked at parquet-mr and I think it always tracks the statistics; I don't have a sense of whether this will materially impact serialization performance (for applications which may not care about the column statistics).